governance/scripts/multisig/info.js

```diff
@@ -5,8 +5,25 @@ const { networks } = require('@unlock-protocol/networks')
 
 const SafeApiKit = require('@safe-global/api-kit').default
 
-const getMultigiInfo = async (chainId, multisig) => {
+const prodSigners = [
```

Comment:
Shouldn't we get these from mainnet directly ?

Reply:
I was wondering about that, but then realized it was going to be weird to check something against itself...